.github/workflows/coverage.yml

```diff
@@ -59,6 +59,6 @@ jobs:
         run: |
           echo "BASE BRANCH CODE COVERAGE is ${{ env.CUR_COVER }}%"
           echo "PULL REQUEST CODE COVERAGE is ${{ env.PR_COVER }}%"
-          if [ "${{ env.PR_COVER }}" -lt "${{ env.CUR_COVER }}"  ]; then
+          if [ "${{ env.PR_COVER }}" -lt "${{ env.CUR_COVER - 2 }}"  ]; then
```

Comment:
what does this do?

Reply:
This means that the PR will only fail if test coverage goes down by more than 2%. I removed it, it is not necessary in this PR.

Reply:
This is broken.
```
The workflow is not valid. .github/workflows/coverage.yml (Line: 59, Col: 14): Unexpected symbol: '-'. Located at position 15 within expression: env.CUR_COVER - 2
```

But I don't think this change is necessary. The code coverage check is not a required build, it does not block PRs. It is really just there to fail as a warning when coverage is dropping.

I think we are better off leaving as is and we can just comment on PRs when it fails outlining why we are dropping coverage or why something is not testable... let me know how this sounds.

Reply:
Ok, I removed this change.